CODEOWNERS

```diff
@@ -6,7 +6,7 @@
 /libs/safekeeper_api/ @neondatabase/safekeepers
 /libs/vm_monitor/ @neondatabase/autoscaling
 /pageserver/ @neondatabase/storage
-/pgxn/ @neondatabase/compute
+/pgxn/ @neondatabase/compute @neondatabase/safekeepers
```

Comment:
/pgxn/neon/ is a better option. And it would be even better if we can require approval only for `walproposer*` files.

Reply:
resolved in https://github.com/neondatabase/neon/pull/7170/commits/c38db0bc8fe8391f55dd1be84bdfbd36072bf638

Reply:
I only added `pgxn/neon/` so that team-safekeeper can have the flexibility of adding new directories/files.